### PR TITLE
fix: send camelCase btnId in link analytics

### PR DESCRIPTION
**Pull request by bmbshlly**

### CODE REVIEW

**Comment by bmbshlly** [comment]
@hardik1003 review

**Comment by bmbshlly** [comment]
@hardik1003 review

File: components/MDXComponents.tsx [hunk 1]
```diff
@@ -1,23 +1,24 @@
 /* eslint-disable import/no-cycle */
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import React from 'react';
+import { camelCase } from 'lodash';
```
Comment (hardik1003) [inline components/MDXComponents.tsx:4]: why not lodash-es?

Comment (bmbshlly) [reply components/MDXComponents.tsx:4]: it does the same thing...using lodash-es type build is causing compiling issues @hardik1003

**Comment by bmbshlly** [comment]
@hardik1003 review

**Review by hardik1003 — APPROVED** [review]

**Pull request merged**